### ISSUE TITLE
https://trilinos.org/ access blocked by Firefox and Chromium (as not secure)

### ISSUE THREAD

**Issue by jschleus** [trilinos/Trilinos#1175]
Half OT: Firefox and Chromium (Linux) block the access to https://trilinos.org/ as "not secure", for e.g. Firefox reports:

```
trilinos.org uses an invalid security certificate.

The certificate is not trusted because it was signed using a signature algorithm
that was disabled because that algorithm is not secure. 
```

**Comment by mhoemmen** [comment]
@trilinos/framework

**Comment by bartlettroscoe** [comment]
I am not seeing a problem on my SNL Windows 7 machine with FireFox 52.0.1.

**Comment by jmgate** [comment]
Seems to be fine on Firefox 52.0 (64-bit) on RHEL7.

**Comment by bmpersc** [comment]
I suspect this has something to do with the fact that google tools are no longer accepting sha1 signed certificates. They've actually been warning about this for a while, but only recently started making it an error. I was able to replicate this on my 10.10 mac with chrome 57.0.2987.110, however, Firefox and safari were both happy to connect.

@jwillenbring maybe we could ask about getting a new cert issued. The cert is set to expire this year anyway so we'll be having to do this soon. However, I'm not 100% sure that a new cert will fix it since our cert seems to be signed with both sha1 and sha256, the later being what google suggests we use. I'll look to see if there is any reason why chrome wouldn't accept a dual signed cert, which is what google has been recommending people get until everything switches to using sha256.

**Comment by jschleus** [comment]
Sorry, forgot to mention I use Firefox 52.0.1 (original 64-bit Linux binary) and Chromium 57.0.2987.98 (64-bit).

**Comment by aprokop** [comment]
Duplicate of #120.
